src/reducers/LeaderboardReducer.js

```diff
@@ -0,0 +1,35 @@
+import _ from 'lodash';
+
+import {
+  FETCH_LEADERBOARD
+} from '../actions/types';
+
+const initialState = {
+  users: []
+};
+
+const LeaderboardReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case FETCH_LEADERBOARD:
```

Comment:
style / semantics preference ->

[use `{}` for each case statement](https://github.com/airbnb/javascript#comparison--switch-blocks)

ie 
```

case FETCH_LEADERBOARD : {
  // code
}
```